docs/sources/tutorials/send-metrics-to-prometheus.md

```diff
@@ -0,0 +1,182 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/tutorials/send-metrics-to-prometheus/
+description: Learn how to send metrics to Prometheus
+title: Use Grafana Alloy to send metrics to Prometheus
+menuTitle: Send metrics to Prometheus
+weight: 15
+---
+# Use Grafana Alloy to send metrics to Prometheus
+In the [Get started with {{< param "PRODUCT_NAME" >}} tutorial][get started], you learned how to configure {{< param "PRODUCT_NAME" >}} to collect and process logs from your local machine and send them to Loki, running in the local Grafana stack. 
+
+As a next step, you will collect and process metrics from the same machine using {{< param "PRODUCT_NAME" >}} and send them to Prometheus, running in the same Grafana stack. 
+
+This process will enable you to query and visualize the metrics sent to Prometheus using the Grafana dashboard.
+
+## Prerequisites
+
+Complete the [previous tutorial][get started] to:
+1. Install {{< param "PRODUCT_NAME" >}} and start the service in your environment.
+1. Set up a local Grafana instance.
+1. Create a `config.alloy` file.
+
+## Configure {{% param "PRODUCT_NAME" %}}
+
+Once the prerequisite steps have been completed, the next step is to configure {{< param "PRODUCT_NAME" >}}.
```

Comment:
This reads as a bit strange to me, since the prerequisites mention following the other tutorial and creating `config.alloy` file, meaning that Alloy is already configured. 

Should this clarify that Alloy is being configured for metric collection?

Reply:
Hi @rfratto ! Thank you for catching that and for all the other great suggestions. 

I accepted all changes and updated the sentence in question to "Once the prerequisite steps have been completed, the next step is to configure {{< param "PRODUCT_NAME" >}} for metric collection."

Please let me know if you would like see additional changes!

Reply:
I think we are good here for this round. I'll merge it as-is and if we need further clarification, we can open a new PR.